1.prep_semantic_comparison.py

```diff
@@ -64,14 +64,14 @@
 files = glob("%s/*.nii.gz" %image_folder)
 
 if len(files) == 0:
-    print("Error, did not find image files in %s. Did you generate them with 0.neurovault_images.py?" %(image_folder))
+    print(("Error, did not find image files in %s. Did you generate them with 0.neurovault_images.py?" %(image_folder)))
 else:
     lookup = dict()
     for f in files:
         image_id = int(os.path.basename(f).strip(".nii.gz"))
         if image_id in concept_df.index:
             lookup[image_id] = f
         else:
-            print("Cannot find image %s in concept data frame" %(image_id))
+            print(("Cannot find image %s in concept data frame" %(image_id)))
```

Comment:
Sorry, that's a side effect of the 2to3 script, no need to keep them 

Sent from my iPhone

> On Aug 14, 2016, at 5:04 PM, Vanessa Sochat <notifications@github.com> wrote:
> 
> In 1.prep_semantic_comparison.py:
> 
> >  else:
> >      lookup = dict()
> >      for f in files:
> >          image_id = int(os.path.basename(f).strip(".nii.gz"))
> >          if image_id in concept_df.index:
> >              lookup[image_id] = f
> >          else:
> > -            print("Cannot find image %s in concept data frame" %(image_id))
> > +            print(("Cannot find image %s in concept data frame" %(image_id)))
> And here!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>